solr/core/src/java/org/apache/solr/handler/admin/CoreAdminHandler.java

```diff
@@ -190,7 +215,12 @@ public void handleRequestBody(SolrQueryRequest req, SolrQueryResponse rsp) throw
 
       // Pick the action
       final String action = req.getParams().get(ACTION, STATUS.toString()).toLowerCase(Locale.ROOT);
-      CoreAdminOperation op = opMap.get(action);
+      CoreAdminOp op = opMap.get(action);
+      log.warn(
+          "action '{}' not found, calling custom action handler. "
+              + "If original intention was to target some custom behaviour "
+              + "use custom actions defined in 'solr.xml' instead",
+          action);
       if (op == null) {
```

Comment:
oops, i could/should have spotted this earlier, sorry. logging needs to be within the `if` block.
```suggestion
      if (op == null) {
      log.warn(
          "action '{}' not found, calling custom action handler. "
              + "If original intention was to target some custom behaviour "
              + "use custom actions defined in 'solr.xml' instead",
          action);
```

Reply:
Oh! Nice catch! Thank you!
Fixed...